terraform/main.tf

```diff
@@ -8,6 +8,9 @@ resource "juju_application" "mlmd" {
   model     = var.model_name
   name      = var.app_name
   resources = var.resources
+  storage_directives = {
+    mlmd-data = "10G"
+  }
```

Comment:
This should be configurable. Could you sync with @misohu so you define this uniformly? I had some live discussions with him as well